committees/maintainer-fund-design-group.md

```diff
@@ -0,0 +1,49 @@
+# Rust Foundation Maintainer Fund Design Committee
+
+## Mission
+
+The *Rust Foundation Maintainer Fund Design* committee is tasked with preparing an RFC that will define how will the funds from the [Rust Foundation Maintainer Fund](https://rustfoundation.org/media/announcing-the-rust-foundation-maintainers-fund/) be used to support Rust Project maintainers.
+
+## Goals
+
+* Gather initial input from stakeholders related to the fund (such as Rust Project maintainers and the Rust Foundation itself).
+* Prepare an RFC proposal that will describe how will the Fund work. Specifically, it should answer questions like:
+    * Which maintainers are eligible for funding from the Fund?
+    * How should we choose the maintainers to fund?
+    * Who will choose the maintainers to fund?
+    * How will we evaluate the work of the funded maintainers?
+* Share the proposal publicly, gather and integrate feedback.
+* Drive the RFC towards being accepted.
```

Comment:
I think we should be making it explicit in some sense that the group and the RFC should error on the side of "something" rather than "perfect" - when the group needs to choose between "let's just make a decision to get something going" and "we need to evaluate all the options in great detail, at the consequence of delaying things", I want the group to be doing the former.